frontend/craco.config.js

```diff
@@ -80,9 +80,7 @@ module.exports = {
         // turn off sourcemaps
         webpackConfig.devtool = "eval"
       } else {
-        // TODO: Trace/Test this for update to GITHUB_ACTION instead
-        const parallel = process.env.CIRCLECI ? false : true
-        minimizerPlugins[terserPluginIndex].options.parallel = parallel
+        minimizerPlugins[terserPluginIndex].options.parallel = true
```

Comment:
Would appreciate an explanation on this part 😅

Reply:
Hm, I think we'll have to just try flipping the flag on and off to see if there's any notable effect that it has on build times 😆

My reasoning for just simplifying without fully knowing what's going on is that the flag has been set to true in GitHub actions since we migrated, so I don't think it'll hurt to just remove this code. My guess for why this was disabled in CircleCI is that parallelization actually hurt performance more than it helped because it added context-switching overhead, and CircleCI runners only have 2 CPU cores. Unsure whether that's true in Github Actions

We may also just be able to get rid of the clause entirely if it defaults to true.

Reply:
Haha sounds like a good plan to me 👍🏼

Reply:
So my very-unscientific testing of running this with parallelization off a few times and comparing the runtimes to some recent commits doesn't show any clear improvement in build times, so I think it's safe to just assume that it's not particularly important either way (it's also possible that, if we're using large runners in GHA for all jobs, that it would be important with normal runners but isn't given the higher-spec machines we're using)